mesonbuild/modules/hotdoc.py

```diff
@@ -415,7 +419,14 @@ def has_extensions(self, state, args, kwargs):
         # --c-include-directories
         KwargInfo(
             'dependencies',
-            ContainerTypeInfo(list, (Dependency, build.StaticLibrary, build.SharedLibrary)),
+            ContainerTypeInfo(list, (Dependency, build.StaticLibrary, build.SharedLibrary,
+                                     build.CustomTarget, build.CustomTargetIndex)),
```

Comment:
For backward compatibility with GStreamer it needs to also accept `build.Executable`. I think it should even be `build.BuildTarget` because that's what accepts `process_dependencies()`.

Reply:
I thought that the executable was determined to simply be a gstreamer bug, because it didn't even do what they wanted?

Reply:
Yes, but since we are doing a PR for backward compat anyway, better support what gst currently does so it keeps working in already released versions.

Reply:
Could always emit a warning if an executable is passed

Reply:
Of course that would need to be added into FeatureDeprecated() below.

Reply:
Let's close this since it's not strictly needed to fix gst build issue.